.travis.yml

```diff
@@ -0,0 +1,16 @@
+# vim ft=yaml
+language: python
+sudo: false
+cache:
+  directories:
+    - $HOME/.cache/pip
+
+python:
+  - 3.6
+
+before_install:
+  - python -m pip install --upgrade pip
+  - pip install "flake8<3.0" flake8-putty
```

Comment:
If we want to use Black instead of flake8 as discussed in #16, then this line would be something like
```
- pip install black
```